lib/suites.js

```diff
@@ -0,0 +1,33 @@
+/*!
+ * Copyright (c) 2018-2022 Digital Bazaar, Inc. All rights reserved.
+ */
+import * as bedrock from '@bedrock/core';
+import {DataIntegrityProof} from '@digitalbazaar/data-integrity';
+import {Ed25519Signature2018} from '@digitalbazaar/ed25519-signature-2018';
+import {Ed25519Signature2020} from '@digitalbazaar/ed25519-signature-2020';
+import {
+  cryptosuite as eddsa2022CryptoSuite
+} from '@digitalbazaar/eddsa-2022-cryptosuite';
+
+// DataIntegriyProof should work for multiple cryptosuites
+const cryptosuites = [
+  eddsa2022CryptoSuite
+];
+
+export function addSuites() {
+  const suite = [];
+  const cfg = bedrock.config['vc-verifier'];
+  const {supportedSuites} = cfg;
+  if(supportedSuites.includes('Ed25519Signature2018')) {
+    suite.push(new Ed25519Signature2018());
+  }
+  if(supportedSuites.includes('Ed25519Signature2020')) {
+    suite.push(new Ed25519Signature2020());
+  }
+  if(supportedSuites.includes('eddsa-2022')) {
+    for(const cryptosuite of cryptosuites) {
```

Comment:
This is weird... it seems we need better alignment here. It seems that:

1. `cryptosuites` should be a map from cryptosuite name => cryptosuite. We should rename it to `SUPPORTED_CRYPTOSUITES` to match convention.
2. We should iterate through `supportedSuites` and see if the value is one of the legacy suites (2018/2020 from above) and add the suite instance for it if so.
3. If the value is not one of the legacy suites, we should look for it in the `SUPPORTED_CRYPTOSUITES` map. If present, we should add a new `DataIntegrityProof` suite for it, if not, we should throw an not supported error causing the app startup process to bail.

Reply:
This isn't exactly what you asked for, but it might be.... cooler looking: https://github.com/digitalbazaar/bedrock-vc-verifier/pull/36/commits/d695eba7a7147ad8bcd637cd74100803e11bbc56